examples/server_setup.yml

```diff
@@ -11,11 +11,11 @@ roles:
     color: 0x8855ee
   player:
     name: Player Role Name
-    color: 0x00bb99
+    color: 0xc2ffe1
```

Comment:
absolutely not, horrific on light mode
![image](https://github.com/user-attachments/assets/17520feb-d127-4f89-bc68-9a3e7f882fa9)

![image](https://github.com/user-attachments/assets/e152955f-2dbf-4b57-bf7d-87ff7acb794a)